### ISSUE TITLE
Runtime error due to tslib not found in latest version

### ISSUE THREAD

**Issue by philrenaud** [amplitude/Amplitude-Node#74]
Deployed our app to Vercel with the latest @amplitude/node installed; received the following:

```
2021-01-30T00:59:37.035Z	undefined	ERROR	Server is not listening Error: Cannot find module 'tslib'
Require stack:
- /var/task/node_modules/@amplitude/identify/dist/src/identify.js
- /var/task/node_modules/@amplitude/identify/dist/src/index.js
- /var/task/node_modules/@amplitude/node/dist/src/nodeClient.js
- /var/task/node_modules/@amplitude/node/dist/src/index.js
- /var/task/__sapper__/build/server/server.js
- /var/task/launcher.js
- /var/runtime/UserFunction.js
- /var/runtime/index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:815:15)
    at Function.Module._load (internal/modules/cjs/loader.js:667:27)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/var/task/node_modules/@amplitude/identify/dist/src/identify.js:3:15)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Module.require (internal/modules/cjs/loader.js:887:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/var/task/node_modules/@amplitude/identify/dist/src/identify.js',
    '/var/task/node_modules/@amplitude/identify/dist/src/index.js',
    '/var/task/node_modules/@amplitude/node/dist/src/nodeClient.js',
    '/var/task/node_modules/@amplitude/node/dist/src/index.js',
    '/var/task/__sapper__/build/server/server.js',
    '/var/task/launcher.js',
    '/var/runtime/UserFunction.js',
    '/var/runtime/index.js'
  ]
}
RequestId: 4fd597ba-c037-440f-bcd5-c5295200fe93 Error: Runtime exited with error: exit status 1
Runtime.ExitError
```

## Possible Solution
We rolled back to 1.1.1 and the issue was resolved.

**Comment by kelvin-lu** [comment]
@philrenaud thanks for the update - we'll resolve this ASAP.

**Comment by kelvin-lu** [comment]
@philrenaud thanks again for the report - this should be resolved in the newest version (1.3.2) of `@amplitude/node` - let me know if you run into similar issues!

**Comment by jooohhn** [comment]
Fixed by 1.3.2
